src/hoc/AppMargin.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box } from '@mui/material'
+import { ReactNode } from 'react'
+
+export enum SizeofAppMargin {
+  MEDIUM = 'medium',
+  LARGE = 'large',
+}
+
+export const AppMargin = ({
+  children,
+  size,
+  ptb,
+}: {
+  children: ReactNode
+  size: SizeofAppMargin
+  ptb?: string | undefined
```

Comment:
the prop is already optional, do we need extra undefined type?

Reply:
removed